src/zjs_ipm.h

```diff
@@ -69,13 +96,14 @@ typedef struct zjs_ipm_message {
     uint32_t error_code;
 
     union {
-        // AIO
+#ifdef BUILD_MODULE_AIO
```

Comment:
Even defining structs that don't get used shouldn't really hurt anything. A few microseconds faster built time but more clutter in the code.